demo/css/link.css

```diff
@@ -1,6 +1,5 @@
 .wmf-wp-with-preview {
   position: relative;
-  font-weight: 400;
   color: inherit;
```

Comment:
Can you also remove this line since it doesn't do anything

Reply:
DONE.